src/components/VInputField/VInputField.vue

```diff
@@ -88,4 +88,8 @@ export default {
 .input-field:focus-within .info {
   @apply text-dark-charcoal;
 }
+
```

Comment:
The `placeholder-dark-charcoal-70` class in this component should be replaced by `placeholder:text-dark-charcoal-70`.

On the other hand, the style below can be replaced by the `placeholder:font-normal` utility too.